app/jobs/application_submission_job.rb

```diff
@@ -4,6 +4,8 @@ class ApplicationSubmissionJob < ApplicationJob
   queue_as :default
 
   def perform(user:, email_template:)
+    return if Feature.ecf_api_disabled?
```

Comment:
Is it worth queueing your new `ApplicationSubmissionMailer` from here to avoid the duplication? or pulling it out entirely and changing the caller to something like:

```
def enqueue_jobs(application)
  ApplicationSubmissionJob.perform_later(user:, email_template:) unless if Feature.ecf_api_disabled?
  SendApplicationSubmissionEmailJob.perform_later(application:, email_template:)
end
```